### ISSUE TITLE
Gradient artifacts

### ISSUE THREAD

**Issue by paulwinex** [TheMrKingent/NeuralUpscale#2]
Hi. I faced with problem when image have some gradient (3d render)
this is original render:
![original](https://user-images.githubusercontent.com/7323236/188272381-d5d4ef76-c05a-45bb-8401-8b688b15de45.png)

After upscale the image has artifacts on gradients
![upscaled1](https://user-images.githubusercontent.com/7323236/188272415-9011edbc-5eb5-45bf-9564-9208f4ed05e9.png)

If i zoom the image in a see the lines on gradient
![lines](https://user-images.githubusercontent.com/7323236/188272443-19d59e91-19ad-40c6-aaca-72e4e85a2561.png)

It is problems of model? Should I train my own model for such case?

Source images: PNG 8bit

**Comment by TheMrKingent** [comment]
Yes, this seems quite a specific task, and the pre-trained models appear to be not very well suited. I actually trained them with the intent of being more general purpose oriented. Definitely you will have better results training your own model based on samples which are closer to what your use case would be.

**Comment by paulwinex** [comment]
Thanks
